apps/blockchain/ethereum/test/Bridge.t.sol

```diff
@@ -600,5 +604,31 @@ contract BridgeTest is Test, IStarklaneEvent {
         assert(payload.length == reqContent.length);
         return (nonce, payload);
     }
+     
+    // here is test   for set L1 L2 collection 
+   
+    function test_SetL1L2CollectionMapping() public {
+    
+        // Check for the event emission
+        vm.expectEmit(true, true, true, true);
+        emit L1L2CollectionMappingUpdated(collectionL1, snaddress.unwrap(collectionL2));
+       IStarklane(bridge).setL1L2CollectionMapping(collectionL1, collectionL2, false);
+    }
+
+    function testInvalidL1Address() public {
```

Comment:
Could you please change the test name to reflect it's checking `setL1L2CollectionMapping`